packages/gatsby/cache-dir/static-entry.js

```diff
@@ -329,6 +333,21 @@ export default (pagePath, callback) => {
     />
   )
 
+  // Add chunk mapping metadata
+  const scriptChunkMapping = `/*<![CDATA[*/window.___chunkMapping=${JSON.stringify(
```

Comment:
add chunk mappings to html (mapping from the chunk names to the actual path of the js file) for component chunks.